tools/scripts/build/build-mobile-curriculum.js

```diff
@@ -33,6 +43,39 @@ exports.buildMobileCurriculum = function buildMobileCurriculum(json) {
     }
   }
 
+  function getBlockDescription(superBlockKey, blockKey) {
+    try {
+      const intros = JSON.parse(fs.readFileSync(blockIntroPath));
+
+      if (superBlockKey == '14-responsive-web-design-22') {
+        superBlockKey = '2022/responsive-web-design';
+      } else if (superBlockKey == '13-relational-databases') {
+        superBlockKey = 'relational-database';
+      }
```

Comment:
The issue here is that `superBlockKey` isn't a key. It's just a subpath. If that changes (and it's likely to) this stops working.

The fix is to build `curriculum.json` with keys instead of subpaths, then this gets a lot simpler.  I'll see what I can do!

Reply:
great thanks